core/api/src/services/notifications/index.ts

```diff
@@ -194,47 +199,34 @@ export const NotificationsService = (): INotificationsService => {
     transaction,
   }: NotificatioSendTransactionArgs): Promise<true | NotificationsServiceError> => {
     try {
-      const settings = await getUserNotificationSettings(recipient.userId)
-      if (settings instanceof Error) {
-        return settings
-      }
-
-      const hasDeviceTokens = settings.pushDeviceTokens.length > 0
-      if (!hasDeviceTokens) return true
+      const request = new HandleNotificationEventRequest()
+      const event = new NotificationEvent();
+      const tx = new TransactionInfo();
 
+      tx.setUserId(recipient.userId);
       const type = getPushNotificationEventType(transaction)
       if (!type) return true
-
+      tx.setType(notificationTypeToProto(type));
+      const settlementAmount = new ProtoMoney();
+      settlementAmount.setMinorUnits(transaction.settlementAmount)
+      settlementAmount.setCurrencyCode(transaction.settlementCurrency);
+      tx.setSettlementAmount(settlementAmount);
       const displayAmountMajor = transaction.settlementDisplayAmount
       const displayCurrency = transaction.settlementDisplayPrice.displayCurrency
-      const displayAmountMinor = roundToBigInt(
-        majorToMinorUnit({ amount: Number(displayAmountMajor), displayCurrency }),
+      const displayAmountMinor = 
```

Comment:
why remove ```roundToBigInt```

Reply:
Because we don't need a `bigint` type... we need a `number` type - they are not compatible.